ingestion/Dockerfile

```diff
@@ -31,7 +32,13 @@ WORKDIR /ingestion
 COPY ingestion /ingestion
 
 ARG INGESTION_DEPENDENCY=all
-RUN pip install ".[${INGESTION_DEPENDENCY}]"
+RUN pip install ".[postgres,mysql,redshift]"
```

Comment:
@ulixius9 Let's update this to be `pip install ".[${INGESTION_DEPENDENCY}]"

Reply:
Opps!